src/map/script.c

```diff
@@ -14492,6 +14493,73 @@ BUILDIN_FUNC(npctalk)
 	return SCRIPT_CMD_SUCCESS;
 }
 
+/*==========================================
+ * chatmes (sends message to npc's waitingroom)
+ *------------------------------------------*/
+BUILDIN_FUNC(chatmes)
+{
+	struct npc_data* nd = NULL;
+	const char* str = script_getstr(st,2);
+
+	if (script_hasdata(st, 3))
+		nd = npc_name2id(script_getstr(st, 3));
+	else
+		nd = (struct npc_data *)map_id2bl(st->oid);
+
+	if (nd != NULL && nd->chat_id) {
+		char message[256];
+		safesnprintf(message, sizeof(message), "%s", str);
+		clif_GlobalMessage(map_id2bl(nd->chat_id), message, CHAT_WOS);
+	}
```

Comment:
I removed it intentionally since this is not includet in npctalk anymore.

If you want to display it to the surrounding area -> npctalk.
If you want to display it in the waitingroom -> chatmes.